pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.609.3</version>
```

Comment:
Requiring our users to upgrade their Jenkins is something I don't want to do without a very good reason to do so: it puts up a barrier to using our plugin. If there is a reason we can't continue using 1.609.3, please list it